webui/index.html.tpl

```diff
@@ -30,6 +30,10 @@
           {{ end }}
         {{ end }}
       {{ end }}
+
+      <span>
+        <a href="{{ startDateParam .Options.TimeRangeMin }}">Previous</a> / <a href="{{ endDateParam .Options.TimeRangeMax }}">Next</a>
+      </span>
```

Comment:
@joshsadler do you have a handy CSS tweak to display this better? It looks like each `a` is put into a box on its own line. 

![Screen Shot 2021-12-06 at 3 48 48 PM](https://user-images.githubusercontent.com/120951/144940807-41a3dd10-3bf3-4eee-9101-9131ea6ba837.png)